regimes/se/tax_categories.go

```diff
@@ -0,0 +1,94 @@
+package se
+
+import (
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/tax"
+)
+
+// Tax rates specific to Sweden.
+const (
+	TaxRateStandard       cbc.Key = "Normalskattesats"
+	TaxRateReduced        cbc.Key = "Skattesats 12 procent"
+	TaxRateHeavilyReduced cbc.Key = "Skattesats 6 procent"
+	TaxRateExempt         cbc.Key = "Momsfri"
+)
+
+var taxCategories = []*tax.CategoryDef{
+	{
+		Code: tax.CategoryVAT,
+		Name: i18n.String{
+			i18n.EN: "VAT",
+			i18n.SE: "Moms",
+		},
+		Title: i18n.String{
+			i18n.EN: "Value Added Tax",
+			i18n.SE: "Moms",
+		},
```

Comment:
Often a good idea to list `Sources` here so that we know where to look up data in the future.